idl/chromadb/proto/coordinator.proto

```diff
@@ -108,6 +111,7 @@ message DeleteCollectionRequest {
   string id = 1;
   string tenant = 2;
   string database = 3;
+  optional bool with_segments = 4;
```

Comment:
should deleting a collection only _optionally__ delete segments? Is there a valid case for wanting to delete collections but leave segments.